pkg/operator/openshift_sdn.go

```diff
@@ -109,3 +102,95 @@ func sdnPluginName(n v1.SDNMode) string {
 	}
 	return ""
 }
+
+// controllerConfig builds the contents of controller-config.yaml
+// for the controller
+func (h *Handler) controllerConfig() (string, error) {
+	c := h.config.Spec.DefaultNetwork.OpenshiftSDNConfig
+
+	// generate master network configuration
+	ippools := []cpv1.ClusterNetworkEntry{}
+	for _, net := range h.config.Spec.ClusterNetworks {
+		ippools = append(ippools, cpv1.ClusterNetworkEntry{CIDR: net.CIDR, HostSubnetLength: net.HostSubnetLength})
+	}
+
+	var vxlanPort uint32 = 4789
```

Comment:
Is 4789 a constant? Should the admin be able to specify a value?

Reply:
It's just the default, but can be overridden.